src/Sylius/Bundle/PromotionBundle/Form/Type/Action/PercentageDiscountConfigurationType.php

```diff
@@ -34,6 +34,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'constraints' => [
                     new NotBlank(),
                     new Type(['type' => 'numeric']),
+                    new Range([
+                        'min' => 0.01,
```

Comment:
We shouldn't prevent from creating e.g. `5,5%` discount. Perhaps we can put `0` as minimum? Or just lower value `0.0001`, but this introduces not necessary limitation.